src/content/docs/en/reference/configuration-reference.mdx

```diff
@@ -268,17 +263,19 @@ You can also set this if you prefer to be more strict yourself, so that URLs wit
 
 <p>
 
-**Type:** `'where' | 'class'`<br />
-**Default:** `'where'`<br />
+**Type:** `'where' | 'class' | 'attribute'`<br />
+**Default:** `'attribute'`<br />
 <Since v="2.4" />
 </p>
 
 Specify the strategy used for scoping styles within Astro components. Choose from:
-  - `'where'` - Use `:where` selectors, causing no specifity increase.
-  - `'class'` - Use class-based selectors, causing a +1 specifity increase.
+  - `'where'` 		- Use `:where` selectors, causing no specifity increase.
+  - `'class'` 		- Use class-based selectors, causing a +1 specifity increase.
+  - `'attribute'` 	- Use `data-` attributes, causing no specifity increase.
 
 Using `'class'` is helpful when you want to ensure that element selectors within an Astro component override global style defaults (e.g. from a global stylesheet).
 Using `'where'` gives you more control over specifity, but requires that you use higher-specifity selectors, layers, and other tools to control which selectors are applied.
+Using 'attribute' is useful when you are manipulating the `class` attribute of elements and need to avoid conflicts between your own styling logic and Astro's application of styles.
```

Comment:
small nit
```suggestion
Using `'attribute'` is useful when you are manipulating the `class` attribute of elements and need to avoid conflicts between your own styling logic and Astro's application of styles.
```